core/src/main/scala/akka/persistence/r2dbc/journal/R2dbcJournal.scala

```diff
@@ -42,7 +42,10 @@ object R2dbcJournal {
  * INTERNAL API
  */
 @InternalApi
-final class R2dbcJournal(config: Config, cfgPath: String) extends AsyncWriteJournal {
+final class R2dbcJournal(
+    config: Config, // FIXME: config: Config not used
```

Comment:
I think it is needed for the Akka Persistence creation of the instance (via config). There are a few variants supported but not all combinations.